tutorials/nservicebus-101/nsb101-intro-paragraph.include.md

```diff
@@ -0,0 +1,3 @@
+The best way to get started with NServiceBus is to use it to build something realistic. In doing so you'll learn the architectural concepts behind the software, and start to learn its capabilities. In this step-by-step guide, you'll be building a back end for a retail e-commerce system. You'll learn how to send asynchronous messages between processes, how to use the Publish/Subscribe pattern to decouple business processes, and the advantages of using reliable messaging to enable automatic retries after processing failures.
+
+The guide is divided into five lessons, each of which can be accomplished in a half hour or less – perfect for your lunch break.
```

Comment:
Would you prefer "I divided the guide into five lessons" ? That would be the active voice.

What would you suggest instead of the em-dash, that incorporates the sentence fragment after it?